kratos/input_output/vtk_output.cpp

```diff
@@ -431,21 +449,30 @@ void VtkOutput::WriteConditionResultsToFile(const ModelPart& rModelPart, std::of
 {
     const auto& r_local_mesh = rModelPart.GetCommunicator().LocalMesh();
     Parameters condition_results = mOutputSettings["condition_data_value_variables"];
+    Parameters condition_flags = mOutputSettings["condition_flags"];
 
     int num_elements = static_cast<int>(r_local_mesh.NumberOfElements());
     rModelPart.GetCommunicator().SumAll(num_elements);
 
     int num_conditions = static_cast<int>(r_local_mesh.NumberOfConditions());
     rModelPart.GetCommunicator().SumAll(num_conditions);
 
-    if (num_elements == 0 && num_conditions > 0) {
-        // write cells header
+    if (num_elements == 0 && num_conditions > 0) { // TODO: Can we have conditions and elements at the same time?
```

Comment:
no, @adityaghantasala explained to me that we have no control over what paraview will display in case there is an element and a condition in the same location

We were discussing of introducing the "WriteConditions" flag same as the GiD output has

Reply:
=> this we can start doing after this is merged, then we can also get rid of the annoying warning

Other than that we also spoke abt a "WriteDeformedConfiguration" flag, same as GiD has